pkg/manager/member/tikv_scaler.go

```diff
@@ -161,6 +161,11 @@ func (s *tikvScaler) ScaleIn(meta metav1.Object, oldSet *apps.StatefulSet, newSe
 			klog.Infof("tikv scale in: set pvc %s/%s annotation: %s to %s",
 				ns, pvcName, label.AnnPVCDeferDeleting, now)
 
+			// endEvictLeader for TombStone stores
+			if err = endEvictLeaderbyStoreID(s.deps, tc, id); err != nil{
```

Comment:
You can just get the ordinal from the Pod name via `GetOrdinalFromPodName`.

Reply:
Sometimes the pod is gone, the operator will be panic.

Reply:
The pod name is just a string, nothing to do with the real Pod.